docs/usage/config-overview.md

```diff
@@ -0,0 +1,209 @@
+# Renovate Configuration Overview
+
+When Renovate runs on a repository, the final config in use is derived from:
+
+- Global config
+- Inherited config
+- Repository config
+- Resolved presets referenced in config
+
+## Types of Config
+
+### Global config
+
+Global config is the term for config defined by the person or team responsible for running the bot.
+Global config can contain config which is "global only" as well as any configuration options which are valid in Inherited config or Repository config.
+
+If you are an end user of Renovate, such as a user of the Mend Renovate App, then you don't need to care too much about global config other than knowing that there are some settings you cannot change because they are global-only.
+You can skip the rest of this "Global config" section and proceed to "Inherited config".
+
+Global config can be read from file, environment, or CLI parameters.
+You must configure at least one of these in order to provide your bot with the minimum details it needs to run, e.g. credentials.
+
+#### File config
+
+The first place Renovate global config is read from is from file.
+By default Renovate checks for the existence of `config.js` in the current working directory, but you can override this by defining `RENOVATE_CONFIG_FILE` in env, e.g. `RENOVATE_CONFIG_FILE=/tmp/my-renovate-config.js`.
+
+By default Renovate allows the config file to be _missing_ and does not error if it cannot find it, however if you have configured `RENOVATE_CONFIG_FILE` but the path you specified is not found then Renovate will error and exit, as it's assumed you have a configuration problem.
+If the file is found but cannot be parsed then Renovate will also error and exit.
+
+Global config files can be `.js` or `.json` files.
+JS files can use synchronous or asyncronous methods inside, including even to fetch config information from remote hosts.
+
+#### Environment config
+
+Global config can be defined using environment variables.
```

Comment:
We probably should mention here that all env variables are prefixed with `RENOVATE_` and link exceptions like `LOG_LEVEL`, `LOG_CONTEXT` or the OTEL variables.

Reply:
Added